rebar.config

```diff
@@ -67,12 +67,12 @@
 ]}.
 
 {relx, [
-    {release, {hpr, "1.0.0"}, [hpr]},
+    {release, {hpr, git}, [hpr, runtime_tools, tools, recon]},
     {vm_args, "config/vm.args"},
-    {sys_config, "config/sys.config"},
     {sys_config_src, "config/sys.config.src"},
-    {include_src, true},
-    {overlay, []},
+    {include_src, false},
```

Comment:
Does this mean we won't be able to hot load?

Reply:
Don't think so, just makes the image size as small as possible - by hot loading do you mean using sasl and systools like `relujp` or just sideloading new beams?

Reply:
> sideloading new beams